tests/sentry/mail/test_actions.py

```diff
@@ -166,6 +167,45 @@ def test_full_integration(self):
         assert sent.to == [self.user.email]
         assert "uh oh" in sent.subject
 
+    @with_feature("organizations:issue-alert-fallback-targeting")
+    def test_full_integration_fallthrough(self):
+        one_min_ago = iso_format(before_now(minutes=1))
+        event = self.store_event(
+            data={
+                "message": "hello",
+                "exception": {"type": "Foo", "value": "uh oh"},
+                "level": "error",
+                "timestamp": one_min_ago,
+            },
+            project_id=self.project.id,
+            assert_no_errors=False,
+        )
+        action_data = {
+            "id": "sentry.mail.actions.NotifyEmailAction",
+            "targetType": "IssueOwners",
+            "fallthroughType": "AllMembers",
+        }
+        condition_data = {"id": "sentry.rules.conditions.first_seen_event.FirstSeenEventCondition"}
+
+        Rule.objects.filter(project=event.project).delete()
```

Comment:
Why do you need to delete rules for the project? Are there some rules already set up as part of `setUp`?
Would you mind making mention it in a comment if that's the case?